### PR TITLE
Fix luke docstring

### PR DESCRIPTION
**Pull request by qqaatw**
# What does this PR do?

The output logits of `LukeForEntitySpanClassification` is in `(batch_size, entity_length, config.num_labels)` shape instead of `(batch_size, config.num_labels)`, which is that of `LukeForEntityClassification`.

https://github.com/huggingface/transformers/blob/8b67f20935e48b26c5803cf31e0e89b9cfaa22ab/tests/models/luke/test_modeling_luke.py#L402-L404

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

@sgugger

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Review by LysandreJik — APPROVED** [review]
Indeed! Sounds good to me, @qqaatw, thank you for your contribution!
